docs/getting-started/form-submissions.md

```diff
@@ -71,6 +71,45 @@ query set as the `q` query parameter in the URL. The `GET` handler will then
 filter the names array based on the query, and pass it to the page component for
 rendering.
 
+## POST request with `multipart/form-data`
+
+Forms can either submit as a GET request with URL search parameter encoding, or as a POST request with `multipart/form-data`.
+
+This example demonstrates how to handle `multipart/form-data` `<form>` submissions:
+
+```tsx
+// routes/subscribe.tsx
+import { Handlers } from "$fresh/server.ts";
+
+export const handler: Handlers = {
+  async GET(req, ctx) {
+    return await ctx.render();
+  },
+  async POST(req, ctx) {
+      const form = await req.formData();
+      const email = form.get("email")?.toString() || null;
```

Comment:
why `|| null` when `.get` signature is `FormDataEntryValue | null` ?

```suggestion
      const email = form.get("email")?.toString();
```

Reply:
I've seen this pattern in a few other examples and picked it up. Not seen anything in docs and I admit, I should've checked the API.

Reply:
ok, no pb. ;)